src/content/tags.liquid

```diff
@@ -10,10 +10,9 @@ permalink: /tags/{{ tag | slugify }}/
 schemaType: webPage
 ---
 
-<h2 class="no-underline">Results for tag “{{ tag }}”</h2>
-
-<ol>
 {% assign taglist = collections[ tag ] %}
+<h2 class="no-underline">{{ taglist.length }} results for tag “{{ tag }}”</h2>
```

Comment:
nit: "results" is incorrectly plural when there's only one result.  (normally I'd say use `ngettext` but I don't know how the templating/l10n is set up here)

Reply:
Good catch. [There is no l10n (yet).](https://github.com/mozilla/extension-workshop/issues/665) 

As this restores [parity with what is in prod](https://extensionworkshop.com/tags/?q=publication), I'll ship it and file a follow-up bug to fix this correctly.

Reply:
Filed here: https://github.com/mozilla/extension-workshop/issues/716